tests/UI/pages/install/index.ts

```diff
@@ -222,7 +343,7 @@ class Install extends CommonPage {
    * @param timeout {number} Time to wait for step to finish
    * @returns {Promise<boolean>}
    */
-  async isInstallationStepFinished(page, step, timeout = 30000) {
+  async isInstallationStepFinished(page: Page, step: string, timeout = 30000): Promise<boolean> {
```

Comment:
```suggestion
  async isInstallationStepFinished(page: Page, step: string, timeout: number = 30000): Promise<boolean> {
```

Reply:
Done, but typescript don't recommend this -> https://typescript-eslint.io/rules/no-inferrable-types/
they all be deleted when adding "plugin:@typescript-eslint/recommended"

Reply:
As you want, but it's less clear on the type :(